source/MaterialXContrib/CMakeLists.txt

```diff
@@ -23,7 +23,8 @@ set_target_properties(
     COMPILE_FLAGS "${EXTERNAL_COMPILE_FLAGS}"
     LINK_FLAGS "${EXTERNAL_LINK_FLAGS}"
     VERSION "${MATERIALX_LIBRARY_VERSION}"
-    SOVERSION "${MATERIALX_MAJOR_VERSION}")
+    SOVERSION "${MATERIALX_MAJOR_VERSION}"
+    DEBUG_POSTFIX "d")
```

Comment:
As with other options, I would make the post-fix a build option to you can use ${MATERIALX_DEBUG_POST} or some such thing. If you need an extra "d" then it can be added .